templates/report.html.jinja

```diff
@@ -265,54 +282,49 @@
 
     <div class="responsive-prose max-w-none">
         <h2>
-            Consistency with the California GTFS Minimum Guidelines
+            Consistency with the California Transit Data Guidelines
         </h2>
-
+        <p>
+            The following checks measure adherence the GTFS Schedule Compliance portion of the California Transit Data Guidelines
```

Comment:
`California Transit Data Guidelines` text is repeated (can remove this instance before the link I think?)

Reply:
also looks like "to" should be added after adherence

Reply:
thanks y'all